components/shared/net/request.rs

```diff
@@ -208,7 +208,15 @@ impl RequestBody {
     }
 
     pub fn len(&self) -> Option<usize> {
-        self.total_bytes.clone()
+        self.total_bytes
+    }
+
+    pub fn is_empty(&self) -> bool {
+        if let Some(v) = self.len() {
+            v == 0
+        } else {
+            false
+        }
```

Comment:
Not all clippy warnings are useful. We should remove this method if it's unused.